acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/EventEmitterAcceptanceTest.java

```diff
@@ -53,21 +47,23 @@ public void setUp() throws Exception {
 
   @Test
   public void shouldDeployContractAndAllowLookupOfValuesAndEmittingEvents() throws Exception {
-    System.out.println("Sending Create Contract Transaction");
     final EventEmitter eventEmitter =
-        EventEmitter.deploy(node.web3j(), MAIN_CREDENTIALS, DEFAULT_GAS_PRICE, DEFAULT_GAS_LIMIT)
-            .send();
+        node.execute(transactions.createSmartContract(EventEmitter.class));
+
     final Observable<StoredEventResponse> storedEventResponseObservable =
         eventEmitter.storedEventObservable(new EthFilter());
+
     final AtomicBoolean subscriptionReceived = new AtomicBoolean(false);
+
     storedEventResponseObservable.subscribe(
         storedEventResponse -> {
           subscriptionReceived.set(true);
           assertEquals(BigInteger.valueOf(12), storedEventResponse._amount);
         });
 
-    final TransactionReceipt send = eventEmitter.store(BigInteger.valueOf(12)).send();
+    final TransactionReceipt receipt = eventEmitter.store(BigInteger.valueOf(12)).send();
 
+    assertNotNull(receipt);
     assertEquals(BigInteger.valueOf(12), eventEmitter.value().send());
     assertTrue(subscriptionReceived.get());
```

Comment:
The assertions here are still not as declarative as I'd ideally like but it's hard to see how to improve it without losing the very handy Java wrappers that web3j provides. Probably have to live with it for now.